PowerShell/JumpCloud Module/Private/Association/Invoke-JCAssociation.ps1

```diff
@@ -150,23 +150,28 @@
                                             {
-                                                $Command_Template_Associations_Targets_Post = 'JumpCloud.SDK.V2\Set-JcSdk{0}Association -{0}Id:("{1}") -Id:("{2}") -Op:("{3}") -Type:("{4}") -AttributeSudoEnabled'
+                                                $attributeDictionary.sudo = @{'enabled' = $true; 'withoutPassword' = $false }
+                                                $Command_Template_Associations_Targets_Post = 'JumpCloud.SDK.V2\Set-JcSdk{0}Association -{0}Id:("{1}") -Id:("{2}") -Op:("{3}") -Type:("{4}") -Attributes:("{5}")'
                                             }
                                             if ($additionalProperties.sudo.withoutPassword -And -Not $additionalProperties.sudo.enabled)
                                             {
-                                                $Command_Template_Associations_Targets_Post = 'JumpCloud.SDK.V2\Set-JcSdk{0}Association -{0}Id:("{1}") -Id:("{2}") -Op:("{3}") -Type:("{4}") -AttributeSudoWithoutPassword'
+                                                $attributeDictionary.sudo = @{'enabled' = $false; 'withoutPassword' = $true }
+                                                $Command_Template_Associations_Targets_Post = 'JumpCloud.SDK.V2\Set-JcSdk{0}Association -{0}Id:("{1}") -Id:("{2}") -Op:("{3}") -Type:("{4}") -Attributes:("{5}")'
                                             }
                                             if ($additionalProperties.sudo.enabled -And $additionalProperties.sudo.withoutPassword)
                                             {
-                                                $Command_Template_Associations_Targets_Post = 'JumpCloud.SDK.V2\Set-JcSdk{0}Association -{0}Id:("{1}") -Id:("{2}") -Op:("{3}") -Type:("{4}") -AttributeSudoEnabled -AttributeSudoWithoutPassword'
+                                                $attributeDictionary.sudo = @{'enabled' = $true; 'withoutPassword' = $true }
+                                                $Command_Template_Associations_Targets_Post = 'JumpCloud.SDK.V2\Set-JcSdk{0}Association -{0}Id:("{1}") -Id:("{2}") -Op:("{3}") -Type:("{4}") -Attributes:("{5}")'
                                             }
                                             if (-Not $additionalProperties.sudo.enabled -And -Not $additionalProperties.sudo.withoutPassword)
                                             {
                                                 $Command_Template_Associations_Targets_Post = 'JumpCloud.SDK.V2\Set-JcSdk{0}Association -{0}Id:("{1}") -Id:("{2}") -Op:("{3}") -Type:("{4}")'
                                             }
-                                            $foundAttributes
+                                            # Return Attributes
+                                            $attributeDictionary
```

Comment:
Return a dotnet dictionary object when we determine what attributes are held by the first user/ system object